db/migrate/20231116154434_add_has_frozen_income_or_assets_to_income_details.rb

```diff
@@ -0,0 +1,7 @@
+class AddHasFrozenIncomeOrAssetsToIncomeDetails < ActiveRecord::Migration[7.0]
+  def change
+    change_table :income_details do |t|
+      t.string :has_frozen_income_or_assets
```

Comment:
Why do we store these as a string (i'm assuming 'yes' or 'no') and not a boolean 🤔

Reply:
Yep all of the Yes no questions are stored as strings and hydrated as "YesNoTypes"